assets/js/search.js

```diff
@@ -182,7 +182,7 @@ module.exports = {
     }
 
     this.searchInput.oninput = () => {
-      if (this.searchInput.value.length < minCharLength) {
+      if (this.searchInput.value.trim().length < minCharLength) {
```

Comment:
Issue was that the route handler for `/suggestions` will trim before validating that at least 3 characters are included.

The client side JS would just check that there were three characters.

So a leading or trailing space will result in failed validation.

The user doesn't really see that error (unless they check the console logs) but it's easily avoidable by just trimming before checking the characters length client side.

This is the root cause of most of the 400 errors we're being alerted about.